galleries/users_explain/axis/index.rst

```diff
@@ -1,6 +1,24 @@
 .. _users_axis:
 
-Controlling and labelling Axis objects
+Controlling and Labelling Axis Objects
 --------------------------------------
 
-Some good material in artist tutorial that should be cribbed from or used here.
+Matplotlib `~.axis.Axis` objects are a type of Artist that
+are the underlying abstraction and the basis for the axes
+on a Matplotlib figure or visualization. Axis objects are
```

Comment:
I think I'm starting to get really confused on the concept of the axis artist now. I'm really struggling on how to describe it further

Reply:
I will re-iterate that I didn't write this section yet because it its very confusing, and indeed I'm strongly considering just dropping it and moving relevant material to Axes.  I would not try and do this as a new contributor unless you have a vision for it.

Reply:
Sorry for the very late reply (I've been extremely busy with another project), but no worries! It was definitely a challenge as a new contributor to try and write this documentation but I understand